homeassistant/components/flexpool/sensors.py

```diff
@@ -0,0 +1,314 @@
+"""Sensor classes for the Flexpool integration."""
```

Comment:
This file can just be combined with `sensor.py`.

Reply:
I thought it'd be cleaner to seperate. I have merged the files.